ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx

```diff
@@ -139,7 +158,20 @@ class ResultsExport extends React.Component<Props, State> {
 
     let response = null
 
-    if (this.state.selectedResults.length > 1) {
+    if (this.props.transformer) {
```

Comment:
```suggestion
    if (this.props.transformer || this.state.selectedResults.length > 1) {
      const cql = getResultSetCql(
        this.state.selectedResults.map((result: Result) => result.id)
      )
      const srcs = Array.from(this.getResultSources())

      response = await exportResultSet(
        this.props.transformer || transformerId,
        {
          cql,
          srcs,
          ...(this.props.transformer ? { args: { transformerId } } : {}),
        }
      )
    }
```

This would let us remove the `else if`, but it might hurt readability.

Reply:
Offline discussion between myself and @andrewkfiedler resulted in agreement that sacrificing readability in this scenario may not be worth it. I'll mark this conversation as resolved unless someone else has an objection. 👍